configure.ac

```diff
@@ -132,8 +132,17 @@ fi
 AC_SUBST(INITIAL_SETUP_CFLAGS)
 AC_SUBST(INITIAL_SETUP_LIBS)
 
-GNOME_COMPILE_WARNINGS([error])
-GNOME_MAINTAINER_MODE_DEFINES
+# -Wsign-compare:
+#   I was motivated to wire up warnings by hours wasted on checking if a
+#   unsigned value is negative, so -Wsign-compare is non-negotiable.
```

Comment:
If `-Wsign-compare` is missing from the default upstream `AX_COMPILER_FLAGS` warning set and you believe it has universal value, please submit a patch [upstream](https://github.com/peti/autoconf-archive) and CC me.